scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -118,9 +144,30 @@
                              :value-set {:set #{val}}
                              nil))))
 
-(defn make-offer-attributes
+(defn make-k8s-attribute
```

Comment:
How does this differ from make-k8s-attribute? This feels like we a should have a common function that both of these call into to make the map.

Reply:
made into common function.